problem_builder/mentoring.py

```diff
@@ -577,32 +546,9 @@ def get_message(self, completed):
             # Student did not achieve a perfect score but can try again:
             return self.get_message_content('incomplete')
 
-    @property
-    def assessment_message(self):
-        """
-        Get the message to display to a student following a submission in assessment mode.
-        """
-        if not self.max_attempts_reached:
-            return self.get_message_content('on-assessment-review', or_default=True)
-        else:
-            return None
-
     @property
     def review_tips(self):
```

Comment:
`review_tips` are also used by `MentoringWithExplicitStepsBlock` so we still return the empty list here.